### PR TITLE
Release v1.30.0/v0.52.0/v0.6.0/v0.0.9

### PR DESCRIPTION
**Pull request by XSAM**
### Added

- Support `OTEL_EXPORTER_OTLP_LOGS_INSECURE` and `OTEL_EXPORTER_OTLP_INSECURE` environments in `go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc`. (#5739)
- The `WithResource` option for `NewMeterProvider` now merges the provided resources with the ones from environment variables. (#5773)
- The `WithResource` option for `NewLoggerProvider` now merges the provided resources with the ones from environment variables. (#5773)
- Add UTF-8 support to `go.opentelemetry.io/otel/exporters/prometheus`. (#5755)
### Fixed

- Fix memory leak in the global `MeterProvider` when identical instruments are repeatedly created. (#5754)
- Fix panic instruments creation when setting meter provider. (#5758)
- Fix panic on instruments creation when setting meter provider. (#5758)
- Fix an issue where `SetMeterProvider` in `go.opentelemetry.io/otel` might miss the delegation for instruments and registries. (#5780)

### Removed

- Drop support for [Go 1.21](https://go.dev/doc/go1.21). (#5736, #5740, #5800)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+<!-- Released section -->
+<!-- Don't change this section unless doing release -->
+
+## [1.30.0/0.52.0/0.6.0/0.0.9] 2024-09-09
+
+This release drops the compatibility guarantee of [Go 1.21].
```
Comment (MrAlias) [inline CHANGELOG.md:16]:
```suggestion
This release drops the compatibility guarantee for [Go 1.21].
```

This seems duplicative of the Removed section entry. Is it needed?

Comment (XSAM) [reply CHANGELOG.md:16]: I am ok to drop this sentence.

Comment (XSAM) [reply CHANGELOG.md]: Done

**Review by MrAlias — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

**Comment by XSAM** [comment]
Waiting https://github.com/open-telemetry/opentelemetry-go/pull/5800 to be merged

**Comment by MrAlias** [comment]
> Waiting #5800 to be merged

Merged.

**Pull request merged**